.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## "Ready for Review" checklist
+
+- [ ] The Vercel preview link has been added to this PR's description
+- [ ] The Asana task has been added to this PR's description
```

Comment:
Do you think it would be acceptable to not include these? As they are pre-defined in the relevant links section, it seems like a potentially unnecessary step.

Reply:
I'd like to keep these in mainly because I've seen in other repos where there are placeholder links for the preview and Asana ticket that come from a PR template that don't get updated with real links, and I think having this checklist right up front will help folks remember to update those links. The placeholder are there primarily so it's clear where they belong. I forget to add the links even after having done it many times, so I also personally find it helpful for these items to be listed. 😊

Reply:
Makes sense to me!